SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Utils/ErrorLogHelper.cs

```diff
@@ -100,6 +100,12 @@ public ErrorLogHelper()
         /// <returns>The most recently modified error log file.</returns>
         public static File GetLastErrorLogFile() => Instance.InstanceGetLastErrorLogFile();
 
+        /// <summary>
+        /// Creates the error storage directory if it does not exist.
```

Comment:
The getter talks only about create, which is unusual for a getter description. I think we need to rework the wording here.

Reply:
```suggestion
        /// Get the error storage directory, or creates it if it does not exist.
```